encode_utils/utils.py

```diff
@@ -266,18 +266,49 @@ def calculate_file_size(file_path):
         raise FileNotFoundError(msg)
     return os.path.getsize(file_path) 
 
-def print_format_dict(dico, indent=2):
+def print_format_dict(dico, indent=2, truncate_long_strings=False):
     """Formats a dictionary for printing purposes to ease visual inspection.
     Wraps the ``json.dumps()`` function.
 
     Args:
         indent: `int`. The number of spaces to indent each level of nesting. Passed directly
             to the ``json.dumps()`` function.
+        truncate_long_strings: `bool`. Defaults to `False`. If `True`, then long strings
+            will be truncated before the object is serialized.
     """
     # Could use pprint, but that looks too ugly with dicts due to all the extra spacing.
+    if truncate_long_strings:
+        dico = truncate_long_strings(dico)
```

Comment:
Since I got a "TypeError: 'bool' object is not callable" here, I guess the variable name should be changed so it doesn't conflict with the name of the method?

Reply:
Yeah I'll change it

Reply:
Changed the method name to `truncate_long_strings_in_objects`